deepnote_toolkit/sql/sql_execution.py

```
@@ -609,13 +628,19 @@ def _cancel_cursor(cursor: "DBAPICursor") -> None:
         pass  # Best effort, ignore all errors
 
 
-def _execute_sql_on_engine(engine, query, bind_params):
+def _execute_sql_on_engine(engine, query, bind_params, setup_statements=None):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_execute_sql_on_engine`

(ANN202)